rm1/src/components/Counter.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+
+//react class component
+export default class Counter extends Component {
+    constructor(){
+        super()
+    }
+
+state = {
+    numberCounter: 0
+}
+
+ IncreaseCount = () => {
+    this.setState({numberCounter : ++this.state.numberCounter});
+};
+
+ DecreaseCount = () => {
```

Comment:
We will be adding a strict ESLint & prettier soon, so do keep in mind indentation and best practices.